src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -202,7 +202,7 @@ public function testParseUnquotedScalarStartingWithReservedIndicator($indicator)
 
     public function getReservedIndicators()
     {
-        return array(array('@'), array('`'));
+        return array(array('@'), array('`'), array('|'), array('>'));
```

Comment:
We support ``>`` too.

IMO, as we were accepting such invalid syntaxes until now, we should only warn about this usage in 2.8, and throw exceptions in 3.0, to give time to people to update their Yaml files